modules/datahandling/userrequests.go

```diff
@@ -234,9 +235,43 @@ func (f userProjectsRequest) process(db dbfs.DBFS) ([]dhClosure, error) {
 			Status: messages.StatusPartialfail,
 			Tag:    f.Tag,
 			Data: struct {
-				Projects []dbfs.ProjectMeta
+				Projects []projectLookupResult
 			}{
-				Projects: projects,
+				Projects: nil,
```

Comment:
hmm, should this be `[]projectLookupResult{}` instead? setting it to nil instead of an empty array might make it go become empty in json rather than an empty array, which might cause issues on the client